app/forms/developers_search_form.rb

```diff
@@ -0,0 +1,22 @@
+class DevelopersSearchForm
+  include Virtus.model
+  include ActiveModel::Model
+  include ActiveRecord::Sanitization::ClassMethods
+
+  attribute :email, String
+  attribute :language, String
+  attribute :programming_language, String
+
+  def search
+    if !email.present? && !language.present? && !programming_language.present?
```

Comment:
@truongle7793 
It will make sense if you do it same below (this is a style of Ruby)

```
return `some thing` if `xxx`
do another case
```

Reply:
@thienhv 
Thanks
I have changed the code to 
```
return `some thing` if `xxx`
```